.travis.yml

```diff
@@ -9,41 +7,73 @@ branches:
     - master
     - develop
     - next
-
     # other targets
-    - truffle-db
+    - trufle-db
+
 
 before_install:
   - npm install -g yarn
+install:
+  - yarn bootstrap
+  - truffle obtain --solc=0.5.8
+
+script: yarn ci
 
-matrix:
+
+jobs:
   fast_finish: true
   include:
     - node_js: 8
       env: FABRICEVM=true
+      dist: xenial
```

Comment:
```suggestion
```

let's also remove the `dist` keys since we want Travis' default (currently `xenial`).